src/makeCalculator.js

```diff
@@ -4,7 +4,33 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: function (num) {
+      return this.result + num;
```

Comment:
[CODE STYLE] - Use arithmetic operators with assignment operator '='. Instead of 'return this.result + num;', use 'this.result += num;'.